tokio/tests/rt_common.rs

```diff
@@ -865,18 +839,18 @@ rt_test! {
             rx.await.unwrap();
         });
 
-        runtime.shutdown_timeout(Duration::from_millis(100));
+        Arc::try_unwrap(runtime).unwrap().shutdown_timeout(Duration::from_millis(100));
     }
 
     #[test]
     fn shutdown_wakeup_time() {
-        let mut runtime = rt();
+        let runtime = rt();
 
         runtime.block_on(async move {
             tokio::time::delay_for(std::time::Duration::from_millis(100)).await;
         });
 
-        runtime.shutdown_timeout(Duration::from_secs(10_000));
+        Arc::try_unwrap(runtime).unwrap().shutdown_timeout(Duration::from_secs(10_000));
```

Comment:
...oh, the `Arc` is because of this, right? Shouldn't we just change the runtime's shutdown methods to do this internally?

Reply:
Honestly, not sure, will tackle shutdown in another PR for now this is fine.